### ISSUE TITLE
Not wanting to play skyblocks version on server but wont let me change 

### ISSUE THREAD

**Issue by HAPPYGUNNER479** [Cazadorsniper/PO3#1531]
# Use the search function before reporting to see if the issue has been reported before.
**Pack Version:**
3.3.61
**Pack Mode (Normal, Titan, Kappa):**
Normal
**Singleplayer, Multiplayer, or LAN?:**
Multiplayer

**Description:**
I have gone into the files and changed the level type and it wont let me play anything but skyblocks version. I want to play in the default world. 
**Steps to reproduce:**

**Crash log (use pastebin or issue will be closed):**

**Comment by Cazadorsniper** [comment]
Are you using a server host company? If not then it should start as default without changing anything and make sure you delete that skyblock world folder.

**Comment by HAPPYGUNNER479** [comment]
yes we are using a server host company. The only way we made it work was while the world was generating I changed the level type to DEFAULT. That's the only way we found it to work. So pretty much change coding while the world was being created. Even after that the level type was still sky blocks but spawned in a default world. So even though coding wasn't right the world was default so I worked but idk for how long.
